inference/core/entities/requests/inference.py

```diff
@@ -18,6 +18,8 @@ class BaseRequest(BaseModel):
     id: str = Field(default_factory=lambda: str(uuid4()))
     api_key: Optional[str] = ApiKey
     start: Optional[float] = None
+    source: Optional[str] = None
+    source_info: Optional[str] = None
```

Comment:
Just want to double check, but if these values are missing or set to null, FastAPI won't fail the validation right?

Reply:
I don't think it will fail. I'll run the image locally and make a few HTTP requests without those fields.

Reply:
Tested that it works with and without those fields sent!